src/main/java/uk/gov/hmcts/darts/task/service/impl/AutomatedTaskServiceImpl.java

```diff
@@ -281,7 +284,8 @@ private void addProcessDailyListToTaskRegistrar(ScheduledTaskRegistrar taskRegis
             automatedTaskRepository,
             lockProvider,
             automatedTaskConfigurationProperties,
-            dailyListProcessor
+            dailyListProcessor,
+            logApi
```

Comment:
Personally as all the automated tasks require the logApi but all use different services, I would have put the logApi before the service parameter as the services will all be dynamic but all will have the same logApi